### ISSUE TITLE
prepare a release 0.6.0

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/MDPOW#4]
* [x] check that everything installs cleanly (eg use a virtualenv)
* [x] change release to TRUE in `__init__.py`
* [x] make release on GitHub (but only once you're sure that the package works)

**Comment by iorga** [comment]
Please make an explicit dependency on PyYAML and check it during the install. After installing PyYAML manually, the install works fine, but the dependency on PyYAML is not checked:
```
iorga@p54:/gem/iorga/repositories/MDPOW > pip install .
Processing /gem/iorga/repositories/MDPOW
    Complete output from command python setup.py egg_info:
    mdpow       : INFO     MDPOW 0.6.0-dev starting.
    mdpow       : INFO     Copyright (c) 2010-2014 Oliver Beckstein and Bogdan Iorga
    mdpow       : INFO     Released under the GNU Public Licence, version 3.
    mdpow       : INFO     Contact the authors for bug reports or questions about licensing.
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-fe59el-build/setup.py", line 11, in <module>
        version = __import__('mdpow').get_version()
      File "mdpow/__init__.py", line 540, in <module>
        import config
      File "mdpow/config.py", line 95, in <module>
        import yaml
    ImportError: No module named yaml
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-fe59el-build
iorga@p54:/gem/iorga/repositories/MDPOW > pip install pyyaml
Collecting pyyaml
  Downloading PyYAML-3.11.tar.gz (248kB)
    100% |████████████████████████████████| 249kB 373kB/s 
Installing collected packages: pyyaml
  Running setup.py install for pyyaml
Successfully installed pyyaml-3.11
iorga@p54:/gem/iorga/repositories/MDPOW > pip install .
Processing /gem/iorga/repositories/MDPOW
  Requirement already satisfied (use --upgrade to upgrade): POW==0.6.0.dev0 from file:///gem/iorga/repositories/MDPOW in /sge/packs/packs_centos/opt/python-2.7.10-CentOS6/lib/python2.7/site-packages
Requirement already satisfied (use --upgrade to upgrade): numpy>=1.0 in /sge/packs/packs_centos/opt/python-2.7.10-CentOS6/lib/python2.7/site-packages (from POW==0.6.0.dev0)
Requirement already satisfied (use --upgrade to upgrade): scipy in /sge/packs/packs_centos/opt/python-2.7.10-CentOS6/lib/python2.7/site-packages (from POW==0.6.0.dev0)
Requirement already satisfied (use --upgrade to upgrade): GromacsWrapper>0.2.5 in /sge/packs/packs_centos/opt/python-2.7.10-CentOS6/lib/python2.7/site-packages (from POW==0.6.0.dev0)
Requirement already satisfied (use --upgrade to upgrade): RecSQL>=0.7.5 in /sge/packs/packs_centos/opt/python-2.7.10-CentOS6/lib/python2.7/site-packages (from POW==0.6.0.dev0)
iorga@p54:/gem/iorga/repositories/MDPOW >
```

**Comment by orbeckst** [comment]
Thanks @iorga --- the explicit yaml dependency was fixed with #6.

**Comment by orbeckst** [comment]
@Becksteinlab/mdpow ,   I am going to make a release today or tomorrow. Everything seems to be working and so we have a good point for a release.

**Comment by orbeckst** [comment]
Just released 0.6.0.
